pkg/driver/node.go

```diff
@@ -383,12 +383,11 @@ func (d *nodeService) NodeUnpublishVolume(ctx context.Context, req *csi.NodeUnpu
 }
 
 func (d *nodeService) NodeGetVolumeStats(ctx context.Context, req *csi.NodeGetVolumeStatsRequest) (*csi.NodeGetVolumeStatsResponse, error) {
-	klog.V(4).Infof("NodeGetVolumeStats: called with args %+v", *req)
 	var resp *csi.NodeGetVolumeStatsResponse
-
 	if req == nil || req.VolumeId == "" {
 		return nil, status.Error(codes.InvalidArgument, "Volume ID not provided")
 	}
+	klog.V(4).Infof("NodeGetVolumeStats: called with args %+v", *req)
```

Comment:
But this wont print if volumeID is an empty string, it fails the purpose, may we can add in the beginning only in the beginning inside null check for that req variable.

Reply:
Fixed as suggested